.gitmodules

```diff
@@ -41,3 +41,9 @@
 [submodule "meta-browser"]
 	path = meta-browser
 	url = git://github.com/OSSystems/meta-browser.git
+[submodule "renesas-rcar-gen3"]
+	path = renesas-rcar-gen3
+	url = https://github.com/slawr/renesas-rcar-gen3.git
```

Comment:
While I'm very happy and eager to get Gen3 support in GDP, I strongly feel that Renesar layer should handle Gstreamer plugins better, so it does not have to completely override the whole plugin packages and assume a very specific version of gstreamer plugins:

https://github.com/slawr/renesas-rcar-gen3/blob/genivi-10/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugins-good_1.4.5.bbappend#L1

If Renesas needs to modify existing gstreamer plugins instead of providing their own, ideally such modifications should be pushed upstream but if that is not possible for any reason, patches against gstreamer plugins in recipes would also be acceptable (at least a lot more cleaner solution than what we have right now).

It would make me very happy as GDP maintainer if this issue is resolved as part of Gen3 addition.